src/py42/clients/cases.py

```diff
@@ -0,0 +1,112 @@
+class CasesClient(object):
+    """A client to expose cases API.
+
+    `Rest documenation <https://default-cases.core-int.cloud.code42.com/swagger-ui.html#/Cases>`__
+    """
+
+    def __init__(self, cases_service, cases_file_event_service):
+        self._cases_service = cases_service
+        self._file_events = cases_file_event_service
+
+    @property
+    def file_events(self):
+        """A collection of method for managing cases file events.
```

Comment:
`A collection of methods for managing cases`

Reply:
Changed.